.travis.yml

```diff
@@ -89,6 +89,17 @@ matrix:
       python: "3.6"
       dist: bionic
 
+    - os: linux
+      sudo: required
+      before_install:
+        - gem install awesome_bot
+      script:
+        - awesome_bot .github/*.md --allow-redirect --allow-dupe --skip-save-results
+        - awesome_bot .github/ISSUE_TEMPLATE/*.md --allow-redirect --allow-dupe --skip-save-results
```

Comment:
These check for broken links inside .md files used for GitHub templates. Caught [here](https://github.com/nexB/scancode-toolkit/commit/d82929ccd197cad0b83697575cc42027dead47b2#r37224758).

Reply:
You could put that directly as comments in the yml. Use # at the start of a line for commenting, like in Python

Reply:
@pombredanne Done!